code/modules/games/cah.dm

```diff
@@ -0,0 +1,262 @@
+// This is a parody of Cards Against Humanity (https://en.wikipedia.org/wiki/Cards_Against_Humanity)
+// which is licensed under CC BY-NC-SA 2.0, the full text of which can be found at the following URL:
+// https://creativecommons.org/licenses/by-nc-sa/2.0/legalcode
+
+//Ported from Polaris;
+//To be rewritten to be made more 'in-universe' and less 'meta-SS13 references', as amusing the latter can be.
+//Pretty sure people in-universe wouldn't exactly see things the same way we SS13 players do -- Mwah
+
+/obj/item/deck/cah
+	name = "\improper CAG deck (white)"
+	desc = "The ever-popular Cards Against The Galaxy word game. This is the white deck."
+	icon_state = "cag_white"
+	var/blanks = 5
+
+/obj/item/deck/cah/black
+	name = "\improper CAG deck (black)"
+	desc = "The ever-popular Cards Against The Galaxy word game. This is the black deck."
+	icon_state = "cag_black"
+	blanks = 0
+
+/obj/item/deck/cah/New()
+	..()
+	var/datum/playingcard/P
+	for(var/cardtext in card_text_list)
+		P = new()
+		P.name = "[cardtext]"
+		P.card_icon = "[icon_state]_card"
+		P.back_icon = "[icon_state]_card_back"
+		cards += P
+	if(!blanks)
+		return
+	for(var/x=1 to blanks)
+		P = new()
+		P.name = "Blank Card"
+		P.card_icon = "[icon_state]_card_back"
+		P.back_icon = "[icon_state]_card_back"
+
+// Black cards.
+/obj/item/deck/cah/black/card_text_list = list(
+	"Why am I itchy?",
+	"Today, police shot ____.",
+	"Our local chaplain is worshipping _____.",
+	"My grandparents ordered a crate full of _____ from the catalogue.",
+	"The FIB was called due to ______.",
+	"Alert! The Sol Navy has armed themselves with _____.",
+	"Order received: ________ is your master.",
+	"Order received: ________ is the enemy.",
+	"_____ blew a hole in the wall.",
+	"Today, researchers found an artefact that made people ____ and ____. (Pick 2)",
+	"There was a rap battle between ____ and ____. (Pick 2)",
+	"Caution, ______ have been detected in collision course with the planet.",
+	"Today's kitchen menu includes _______.",
+	"What did space pirates want when they attacked our home?",
+	"I think our boss is insane. He just demanded ______ in his office.",
+	"Fuckin' scientists, they just turned the entire population of Scottsdan into _______ .",
```

Comment:
Replace Scottsdan here.

Reply:
May I have the reasoning?

Reply:
Nevermind it, forgot it was a place in Biesel.